README.md

```diff
@@ -1,147 +1,94 @@
+# Machine Learning Benchmarks
```

Comment:
@aepanchi please review.
Unfortunately, I can not add you to the reviewers list as usual.